src/mixins/filing-template-mixin.ts

```diff
@@ -432,98 +436,90 @@ export default class FilingTemplateMixin extends Vue {
 
     this.setProvisionsRemoved(filing.alteration.provisionsRemoved)
 
-    // Set Office Addresses
-    this.setOfficeAddresses(businessSnapshot[2])
+    // Store office addresses
+    // *** TODO: should restore office addresses from alteration, not snapshot???
+    this.setOfficeAddresses(businessSnapshot.incorporationAddress)
 
-    // Set Business Contact
-    const contact = {
+    // Store people and roles
+    // *** TODO: should restore people and roles from alteration, not snapshot???
+    this.setPeopleAndRoles(
```

Comment:
Who was working on people and roles? Please see TODO comment (and please respond here).